lib/options.go

```diff
@@ -285,6 +285,8 @@ type Options struct {
 
 	// Redirect console logging to a file
 	ConsoleOutput null.String `json:"-" envconfig:"K6_CONSOLE_OUTPUT"`
+
+	Nic null.String `json:"nic" envconfig:"nic"`
```

Comment:
I'm not sure this envconfig declaration is correct or if it should be "K6_NIC". In any case setting `K6_NIC` or `nic` didn't have an effect in my local tests. Hopefully this wouldn't be difficult to get working (despite of #883), as it would be convenient to set this as an env var.

Also, should we add `omitempty` to the `json` tag? This field will most of the time be null.

Reply:
Agreed, on both accounts. In e6915ddd4d8cbb738e095b4f40a46dc93dc34987, I added `omitempty` and renamed to `K6_NIC` for consistency. Thanks!

Reply:
I tested the environment variable and it seems to be working correctly now.

`env K6_NIC=ens192 ~/go/bin/k6 run ip-test.js`

I confirmed by checking the server access logs on the receiving end of this test. The IPs were random for each VU 👍